### ISSUE TITLE
Add Postgres.js

### ISSUE THREAD

**Issue by colinhacks** [edgedb/imdbench#44]
https://github.com/porsager/postgres

**Comment by porsager** [comment]
I've finally gotten around to give it a shot 😊 Looks to be the fastest 🎉 https://porsager.github.io/imdbench/sql.html

**Comment by elprans** [comment]
Not bad!  Though keep in mind that neither EdgeDB nor asyncpg have native macOS arm64 builds yet, so there's probably some emulation overhead.  Would be interesting to see the results on an amd64 linux box.

Can you submit a pull request with your implementation?

**Comment by porsager** [comment]
Yeah would also be great to see a run on a clean(linux) box. God knows what my mac could be doing while this is running 😝
